code/src/components/GameScreen.js

```diff
@@ -0,0 +1,60 @@
+import React from 'react'
+import { useSelector, useDispatch } from 'react-redux'
+import styled from 'styled-components/macro'
+
+import { games, CarryOnGame } from '../reducers/games'
+import { Container } from '../styling/GlobalStyling'
+import EndScreen from './EndScreen'
+
+const GameScreen = () => {
+  const actions = useSelector(store => store.games.gameStatus.actions)
+  const history = useSelector(store => store.games.history)
+
+  const dispatch = useDispatch()
+
+  const onGoBack = () => {
+    dispatch(games.actions.setPreviousGameStatus())
+  }
+
+  if (actions?.length === 0) {
+    return (
+        <EndScreen />
+    )
+  }
+
+  return (
+    <Container>
+        {actions?.map( (action, index) => (
+            <GameDescriptionContainer key={index}>
+              <GameScreenText>{action.description}</GameScreenText>
+              <button
+                  className="nes-btn"
+                  onClick={() => dispatch(CarryOnGame(action.direction))}>
+                      {action.direction}
+              </button>
+            </GameDescriptionContainer>
+          )
+        )}
+      <button
+        className="nes-btn"
+        disabled={!history.length} 
```

Comment:
The button is not disabled in the first description. Maybe try this instead: history.length < 1?
Right now you end up showing a blank box when pressing go back from the first description.